scripts/create_project_table.py

```diff
@@ -0,0 +1,84 @@
+"""
+Process README files to extract metadata, generate a table of content, 
+and update a central README file with aggregated data.
+"""
```

Comment:
Tiny cosmetic change. It's better to have a single line in the docstring, if possible.

```suggestion
"""Extract metadata from project README files to update table in repo's README file."""
```